pipelines/src/pipelines/tensorflow/training/pipeline.py

```diff
@@ -39,6 +39,7 @@ def tensorflow_pipeline(
     timestamp: str = "2022-12-01 00:00:00",
     staging_bucket: str = os.environ.get("VERTEX_PIPELINE_ROOT"),
     pipeline_files_gcs_path: str = os.environ.get("PIPELINE_FILES_GCS_PATH"),
```

Comment:
create `{PIPELINE_FILES_GCS_PATH}/assets` dynamically in `env.sh` from `ADDITIONAL_SUFFIX`?

Reply:
Good call! So much cleaner now 💆‍♂️ahh